incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -21,6 +21,10 @@ env:
   # used to authenticate app daemon to your home assistant instance
   DASH_URL: ""
   # url for the appdaemon dashboard, an alternative interface for home assistant
+  TZ: "America/Chicago"
+  LATTITUDE: "94"
+  LONGITUDE: "94"
```

Comment:
try floats here, not strings.
like 94.0 without quotes.

Reply:
Thanks for the suggestion. I'll give this a try on my lunch break in a bit.